src/main.cpp

```diff
@@ -5269,6 +5269,19 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
         State(pfrom->GetId())->fPreferHeaders = true;
     }
 
+    // Processing this message type for statistics purposes only, BU currently doesn't support CB protocol
+    // Ignore this message if sent from a node advertising a version earlier than the first CB release (70014)
+    else if (strCommand == "sendcmpct" && pfrom->nVersion >= 70014)
```

Comment:
let's get this properly defined as a NetMsgType.  Even if we don't support it, since it exists it should be in the definitions.

Reply:
Fixed.

I had purposely not created a `NetMsgType` because it's not supported, but that's sufficiently covered by the comments.  Also made sure the new message is showing up in the stats collection
`getstat net/recv/msg/sendcmpct`